### PR TITLE
891 temporary token errors in app

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- Compare netloc to netloc, ignore http or https
This will fix errors where people haven't set `IS_BEHIND_PROXY`

## Checklist
- [ ] All querysets/queries filter by Team (if applicable)
- [ ] Backend tests (if applicable)
- [ ] Cypress E2E tests (if applicable)

### CODE REVIEW

**Review by EDsCODE — CHANGES_REQUESTED** [review]
Few small things

File: posthog/api/test/test_action.py [hunk 1]
```diff
@@ -131,6 +131,13 @@ def test_create_from_other_domain(self, patch_delay):
         }, content_type='application/json', HTTP_ORIGIN='https://somewebsite.com')
         self.assertEqual(response.status_code, 200, response.json())
 
+    # This case happens when someone is running behind a proxy, but hasn't set `IS_BEHIND_PROXY`
+    def test_http_to_https(self, patch_delay):
+        response = self.client.post('/api/action/', data={
+            'name': 'user signed up again',
+        }, content_type='application/json', HTTP_ORIGIN='https://testserver/')
```
[the remark this answers is not in the document]
Comment (timgl) [reply posthog/api/test/test_action.py:138]: Yep

**Comment by timgl** [comment]
spillover fixed

**Review by EDsCODE — APPROVED** [review]

**Comment by janbaykara** [comment]
This has broken the custom action creator on multiple heroku deploys I'm working with. Investigating...

**Comment by EDsCODE** [comment]
@janbaykara Hi, thanks for the heads up! Is the action creator not appearing now or are you hitting the AuthError`No temporary_token set...` when trying to create an action from it

**Comment by timgl** [comment]
@janbaykara Just wanted to let you know that we've fixed this in #909. Let me know if you run into any other issues.

**Pull request merged**